tests/testthat/test-constructor.R

```diff
@@ -57,5 +57,7 @@ test_that("can generate constructor for inherited abstract classes", {
     foo1 <- new_class("foo1", abstract = TRUE, properties = list(x = class_double))
     new_constructor(foo1, list())
     new_constructor(foo1, as_properties(list(y = class_double)))
+    child <- new_class("child", foo1, properties = list(y = class_double))
+    child(y = 0.5)
```

Comment:
Given that the output of this operation is known (i.e. you can test the values of the properties), I think this would be better outside of the snapshot test.

Reply:
Previously, the tests were not failing because the test was seeing if the internal `new_constructor` function was "working", but it never tested if the class could be made.

I could move this outside the snapshot

Reply:
I agree that you should test actually constructing the objects, I just think you should do it outside of the snapshot.